snapshotter/benchmark_aws.yml

```diff
@@ -0,0 +1,122 @@
+AWSTemplateFormatVersion: "2010-09-09"
+
+Description: >
+  This templates spin ups an EC2 instance with EBS volumes suitable for containerd snapshotters benchmarking.
+  The template will create three EBS volumes (/dev/sdb, /dev/sdc, and /dev/sdd) with same performance characteristics.
+  /dev/sdd volume will be used for device mapper thin-pool device.
+
+Parameters:
+  Key:
+    Type: AWS::EC2::KeyPair::KeyName
+    Description: SSH key to use
+
+  AMI:
+    Type: AWS::EC2::Image::Id
+    Description: AMI ID to use for the EC2 instance. Must be RHEL or CentOS
+    Default: "ami-3ecc8f46"
```

Comment:
This AMI is vended through the AWS Marketplace and would require a (free?) subscription in order to launch successfully.  It's also somewhat old ("CreationDate": "2018-06-13T15:58:14.000Z") and I wonder if there's a newer one that we can use.